src/Raven.Server/Monitoring/Snmp/Objects/Server/1.15/TcpActiveConnections.cs

```diff
@@ -0,0 +1,25 @@
+using Lextm.SharpSnmpLib;
+using Sparrow.Server.Extensions;
+
+namespace Raven.Server.Monitoring.Snmp.Objects.Server
+{
+    public class TcpActiveConnections : ScalarObjectBase<Gauge32>
+    {
+        public TcpActiveConnections()
+            : base(SnmpOids.Server.TcpActiveConnections)
+        {
+        }
+
+        protected override Gauge32 GetData()
+        {
+            var properties = TcpExtensions.GetIPGlobalPropertiesSafely();
```

Comment:
Can return null

Reply:
Every extension method that I've added checks that.